e2e/next/src/next.test.ts

```diff
@@ -134,11 +135,11 @@ describe('Next.js Applications', () => {
       checkExport: false,
     });
 
-    // public and shared assets should both be copied to dist
-    checkFilesExist(
-      `dist/apps/${appName}/public/a/b.txt`,
-      `dist/apps/${appName}/public/shared/ui/hello.txt`
-    );
+    // declared assets should be copied to dist
+    checkFilesExist(`dist/apps/${appName}/.next/shared/ui/hello.txt`);
+
+    // undeclared asset should not be copied to dist
+    checkFilesDoNotExist(`dist/apps/${appName}/.next/a/b.txt`);
```

Comment:
Public needs to go to `dist/apps/${appName}/public/a/b.txt` still, or else next will not find it.